test/system/src/main/java/io/pravega/test/system/framework/services/docker/BookkeeperDockerService.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.system.framework.services.docker;
+
+import com.spotify.docker.client.messages.ContainerConfig;
+import com.spotify.docker.client.messages.mount.Mount;
+import com.spotify.docker.client.messages.swarm.ContainerSpec;
+import com.spotify.docker.client.messages.swarm.EndpointSpec;
+import com.spotify.docker.client.messages.swarm.NetworkAttachmentConfig;
+import com.spotify.docker.client.messages.swarm.PortConfig;
+import com.spotify.docker.client.messages.swarm.ResourceRequirements;
+import com.spotify.docker.client.messages.swarm.Resources;
+import com.spotify.docker.client.messages.swarm.RestartPolicy;
+import com.spotify.docker.client.messages.swarm.ServiceMode;
+import com.spotify.docker.client.messages.swarm.ServiceSpec;
+import com.spotify.docker.client.messages.swarm.TaskSpec;
+import java.net.URI;
+import lombok.extern.slf4j.Slf4j;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import static io.pravega.test.system.framework.Utils.DOCKER_NETWORK;
+
+@Slf4j
+public class BookkeeperDockerService extends DockerBasedService {
+
+    private static final int BK_PORT = 3181;
+    private final long instances = 3;
+    private final double cpu = 0.1;
+    private final double mem = 1024.0;
+    private final URI zkUri;
+
+    public BookkeeperDockerService(String serviceName, final URI zkUri) {
+        super(serviceName);
+        this.zkUri = zkUri;
+    }
+
+    @Override
```

Comment:
No need to override if we are just using the super method, this will reduce the amount of code ...

Reply:
removed.